microovn/ovn/bootstrap.go

```diff
@@ -89,7 +89,7 @@ func Bootstrap(s *state.State) error {
 		s,
 		fmt.Sprintf("--db=unix:%s", nbDB),
 		"set-connection",
-		"ptcp:6641",
+		"ptcp:6641,[::]",
```

Comment:
Have not checked, but would we not need to also specify connection type and port for the v6 listener? Something like `ptcp:6641:[::]`

Reply:
Yeah I think it should be colon not comma between port and address.

Reply:
So the comma is to list multiple connections, the format for each connection is defined here: https://manpages.ubuntu.com/manpages/kinetic/man7/ovsdb.7.html

I'll check if `set-connection` actually accepts multiple, or if we need to add them in a different way

Reply:
Yeah we just dont need the comma. So `ptcp:6641:[::]"` rather than `ptcp:6641,[::]"

Reply:
Updated